ledger-api/README.md

```diff
@@ -38,27 +34,24 @@ sbt test
 
 # Release
 
-The release process on jenkins also publishes the [Ledger API Introduction](docs/ledger-api-introduction) documentation package with the release version.
-
 1. Create a branch `ledger-api-release-<major>-<minor>-<point>` for the release. Check the file [ledger-api/version.sbt](version.sbt). For example, let's say the the current
 version of the project is `0.9.3-SNAPSHOT`. You need to create a branch called `ledger-api-release-0-9-3`;
-2. Generate the protobuf ProtoDocs with `ledger-api/docs/ledger-api-introduction/generate-protodocs` and commit to that branch.
-3. Bump the version for the NPM distribution channel:
+2. Bump the version for the NPM distribution channel:
 
       (cd npm-package && npm version <major>.<minor>.<point>)
 
-4. Move the ## Unreleased section of [ledger-api/UNRELEASED.md](UNRELEASED.md) to the new release
+3. Move the ## Unreleased section of [ledger-api/UNRELEASED.md](UNRELEASED.md) to the new release
 version you have created in [ledger-api/CHANGELOG.md](CHANGELOG.md) and commit to that branch. The message of the commit is not important as it
 will be discarded;
-5. Run `sbt release`. This will ask for the release version and the next version, and will create
+4. Run `sbt release`. This will ask for the release version and the next version, and will create
```

Comment:
we don't use SBT anymore

Reply:
Does this also apply to the `Build` section?

Reply:
yes